test_echo.py

```diff
@@ -0,0 +1,13 @@
+from echo_client import client
+
+
+def test_1():
+    assert client('This is a unicode test') == 'This is a unicode test'
+
+
+def test_2():
+    assert client('This string is larger than my current buffer size. It should return all of the characters. This is the last sentence of this test.') == 'This string is larger than my current buffer size. It should return all of the characters. This is the last sentence of this test.'
+
```

Comment:
This line of code is ridiculously over long.  Try to stick to the 79 column width of the pep8 standard.